gui/src/3D/controls/touchHolographicButton.ts

```diff
@@ -264,13 +264,13 @@ export class TouchHolographicButton extends TouchButton3D {
 
     // Mesh association
     protected _createNode(scene: Scene): TransformNode {
-        const collisionMesh = BoxBuilder.CreateBox(this.name ?? "TouchHolographicButton", {
+        const collisionMesh = BoxBuilder.CreateBox((this.name ?? "TouchHolographicButton") + "_CollisionMesh", {
             width: 1.0,
             height: 1.0,
             depth: 1.0,
         }, scene);
         collisionMesh.isPickable = true;
-        collisionMesh.isVisible = false;

```

Comment:
Kind of misleading that isVisible influences the picking logic, but visibility does not. @RaananW Do you know if there is any underlying reason for having these two similar-sounding values perform completely different tasks?

Reply:
it similar in all other tech, that alpha value dont influence picking as it gives possibility to create so called blind buttons that are not visible but can react to inputs, and if element is not visible at all then it dont react to inputs any more

Reply:
In HTML is would be the difference between display hidden and visibility (or even opacity). The element can be there but invisible (with a rate of invisibility), or not visible at all